.github/workflows/composer.yml

```diff
@@ -0,0 +1,26 @@
+name: Composer
+
+on:
+  push:
+    branches: [ master, next ]
+  pull_request:
+    branches: [ master, next ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Copy .env
+      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+    - name: Execute tests (Unit and Feature tests) via PHPUnit
+      env:
+        TWILIO_ACCOUNT_SID: ACXXXX
+        TWILIO_AUTH_TOKEN: xxxxxx
+        TWILIO_NUMBER: phone_number
+      run: |
+        bin/phpunit tests
```

Comment:
By the guidelines, we should be enforcing the use of a linter
https://github.com/twilio-labs/code-exchange/tree/master/guidelines#2-use-a-linter
See an example in the template
https://github.com/TwilioDevEd/sample-template-laravel/blob/master/composer.json#L66-L68
https://github.com/TwilioDevEd/sample-template-laravel/blob/master/.github/workflows/laravel.yml#L36-L37

Reply:
Linter will be added in a different PR